libnavigation-core/src/test/java/com/mapbox/navigation/core/reroute/MapboxRerouteControllerTest.kt

```diff
@@ -382,6 +382,24 @@ class MapboxRerouteControllerTest {
         }
     }
 
+    @Test
```

Comment:
Could you please also add a test for https://github.com/mapbox/mapbox-navigation-android/pull/6650#discussion_r1039281321?

Reply:
Added.